src/headers.py

```diff
@@ -178,7 +178,11 @@ def header_reflect_top_array(obj, prefix, header):
     c_typ = helpers.get_prefixed_pointer(obj.name, obj.subtyp, prefix) or \
         helpers.get_map_c_types(obj.subtyp)
     if obj.subtypobj is not None:
-        c_typ = helpers.get_name_substr(obj.name, prefix) + " *"
+        if obj.doublearray and obj.subtypname is not None:
+            print(obj.__dict__)
```

Comment:
do we need this print or was it used for debugging?

Reply:
sorry, i forget....

Reply:
fixed